demos/common/python/models/utils.py

```diff
@@ -42,6 +42,32 @@ def __init__(self, xmin, ymin, xmax, ymax, score, id, landmarks_x, landmarks_y):
             self.landmarks.append((x, y))
 
 
+class OutputTransform:
+    def __init__(self, input_size, output_resolution):
+        self.output_resolution = output_resolution
+        self.new_resolution = self.compute_resolution(input_size)
+
+    def compute_resolution(self, input_size):
+        self.input_size = input_size
+        size = self.input_size[::-1]
+        self.scale_factor = min(self.output_resolution[0] / size[0],
+                                self.output_resolution[1] / size[1])
+        return self.scale(size)
+
+    def resize(self, image):
```

Comment:
I suggest implement case without resizing/rescaling. It could remove 
```
if transform:
    transform.scale()
```
blocks from user code.

Reply:
Done